acoular/fastFuncs.py

```diff
@@ -790,6 +792,7 @@ def _psf_Formulation4AkaTrueLocation(
 )
 def damasSolverGaussSeidel(A, dirtyMap, nIterations, relax, damasSolution):
     """Solves the DAMAS inverse problem via modified gauss seidel.
```

Comment:
```suggestion
    """
    Solves the DAMAS inverse problem via modified gauss seidel.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`